src/System.Reflection.Primitives/src/System/Reflection/EventAttributes.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+////////////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////
+//
+// EventAttributes are an enum defining the attributes associated with
+// 
+
```

Comment:
Weird formatting. Personally I'd just condense this to two-line regular comment above the class.